admin/src/pages/SettingsPage/index.tsx

```diff
@@ -311,9 +311,10 @@ const SettingsPage = () => {
                               {orderBy(values.selectedContentTypes).map(uid => {
                                 const contentType = allContentTypes.find(item => item.uid == uid);
                                 if (!contentType) return;
-                                const { attributes, info: { displayName }, available, isSingle } = contentType;
-                                const stringAttributes = Object.keys(attributes).filter(_ => STRING_ATTRIBUTE_TYPES.includes(attributes[_].type));
-                                const relationAttributes = Object.keys(attributes).filter(_ => RELATION_ATTRIBUTE_TYPES.includes(attributes[_].type));
+                                const { info: { displayName }, available, isSingle } = contentType;
+                                const attributeKeys = Object.keys(contentType.attributes).sort();
+                                const stringAttributes = attributeKeys.filter(key => STRING_ATTRIBUTE_TYPES.includes(contentType.attributes[key].type));
+                                const relationAttributes = attributeKeys.filter(key => RELATION_ATTRIBUTE_TYPES.includes(contentType.attributes[key].type));
```

Comment:
this can be extracted to separate function

Reply:
All refactors to this file are done on a parallel branch. This branch (since it will be merged to master) is only meant to be a batch of quick fixes for version 2.2.x.